examples/layers/raw/canvas.dart

```diff
@@ -90,6 +101,10 @@ ui.Picture paint(ui.Rect paintBounds) {
   paint.drawLooper = builder.build();
   canvas.drawCircle(ui.Point.origin, radius, paint);
 
+  // When we're done issuing painting commands, we end the recording an receive
+  // a Picture, which is an immutable record of the commands we've issued. You
+  // can draw a Picture into another canvas or include it as part of a
+  // composited scene.
   return recorder.endRecording();
 }
 
```

Comment:
This example should be simpler. I would skip the device pixel ratio transform. (This will temporarily make this example more complicated until we fix #1360.)